CoreFoundation/Parsing.subproj/CFXMLInterface.c

```diff
@@ -168,6 +176,9 @@ _CFXMLInterfaceSAXHandler _CFXMLInterfaceCreateSAXHandler() {
     saxHandler->externalSubset = (externalSubsetSAXFunc)__CFSwiftBridge.NSXMLParser.externalSubset;
     
     saxHandler->initialized = XML_SAX2_MAGIC; // make sure start/endElementNS are used
+#else
+    memset(&saxHandler, 0, sizeof(saxHandler));
```

Comment:
Doesn't the calloc above already promise us zeroed memory?

Reply:
Agreed, this line is not needed, I'll remove it.